server/graphql/playground.html

```diff
@@ -20,8 +20,8 @@
     }
   </style>
 
-  <link rel="stylesheet" href="https://unpkg.com/graphiql@4.0.2/graphiql.min.css" />
-  <link rel="stylesheet" href="https://unpkg.com/@graphiql/plugin-explorer/dist/style.css" />
+  <link rel="stylesheet" href="https://unpkg.com/graphiql@4.0.5/graphiql.min.css" />
+  <link rel="stylesheet" href="https://unpkg.com/@graphiql/plugin-explorer@4.0.4/dist/style.css" />
```

Comment:
This is the latest version of both of these, and they seem to play nicely together